src/Compilers/CSharp/Test/Semantic/Diagnostics/DiagnosticAnalyzerTests.cs

```diff
@@ -1126,80 +1126,6 @@ public void TestReportingDiagnosticWithInvalidSpan()
                 );
         }
 
-        [Fact, WorkItem(13120, "https://github.com/dotnet/roslyn/issues/13120")]
```

Comment:
Looking at this issue, it seems that the runtime check was preventing VS crash.

After removing this, if someone ignored the RSxxxx rule that's going to be implemented in roslyn-analyzers - or using a version without this rule -, will that crash VS?

We should make sure VS doesn't crash.

Reply:
> We should make sure VS doesn't crash.

Analyzers run out-of-process now, which mitigates the IDE crash.